src/main/java/com/karankumar/bookproject/ui/goal/ReadingGoalView.java

```diff
@@ -229,8 +230,8 @@ String calculateProgress(int booksToReadThisYear, int booksReadThisYear) {
         if (booksStillToRead <= 0) {
             schedule = TARGET_MET;
         } else {
-            int weekOfYear = getWeekOfYear();
-            int weeksLeftInYear = weeksLeftInYear(weekOfYear);
+            int weekOfYear = dateUtils.getWeekOfYear();
+            int weeksLeftInYear = dateUtils.getWeeksLeftInYear(weekOfYear);
             double booksStillToReadAWeek = Math.ceil((double) booksStillToRead / weeksLeftInYear);
```

Comment:
`getWeeksLeftInYear` can be invoked through static reference?

Reply:
I have changed the approach to remove the static keyword.